public-ui/src/app/views/quick-search/quick-search.component.ts

```diff
@@ -19,6 +19,13 @@ import {DomainInfosAndSurveyModulesResponse} from '../../../publicGenerated/mode
         './quick-search.component.css']
 })
 export class QuickSearchComponent implements OnInit, OnDestroy {
+    dbLogo = '/assets/db-images/Data_Browser_Logo.svg';
+    dbSubTitle = 'The Data Browser provides interactive views of the publically available ' +
+      'All of Us (AoU) Research Program participant data. Currently, participant provided ' +
```

Comment:
Not sure who wrote the text here, but I would probably recommend that we not use the `AoU` acronym anywhere in the app, even if it's defined as it is here.